synchronization/src/androidTestCyface/java/de/cyface/synchronization/UploadProgressTest.java

```diff
@@ -167,6 +168,9 @@ public void testUploadProgressHappyPath() throws CursorIsNullException {
             insertPoint3d(directionsFile, 1501662636030L, 7.65, -32.550003, -71.700005);
             insertPoint3d(directionsFile, 1501662636050L, 7.65, -33.15, -71.700005);
 
+            // Mark measurement as finished
+            persistenceLayer.setStatus(measurementIdentifier, MeasurementStatus.FINISHED);
```

Comment:
Deswegen lief der test schon länger nicht mehr.
Da der Test als Flaky markiert ist, beschwert sich das CI jedoch nicht, habe es immer nur lokal gesehen und vorerst ignoriert, weil das kein Kunde nutzt.

**Anmerkung:**
Dieser Test hier nutzt, obwohl er MocketHttpConnection verwendet, die echte API.

Reproduzierbar: Per Emulator ausführen, einmal mit Internetverbindung, einmal ohne. Zweites ist Fehlerhaft da Upload nicht erfolgreich.

Um das ganze noch hässlicher zu machen: Wir haben andere Sync AndroidTests, bei denen das genauso ist, die jedoch unfairerweise nicht als Flaky markiert sind.

Ist jetzt nicht dringend, aber sollte irgendwann mal gefixt werden. Habe einen Task erstellt: MOV-683

Das gehört aber jetzt hier nicht in den PR rein, da das kein Priorität hat und den PR nur zeitlich + vom Umfang her aufbläht